kickoff/static/model/release.js

```diff
@@ -0,0 +1,129 @@
+var REGEXES = {
+    beta: /^\d+\.[\d.]+b\d+(build\d+)?$/, // Examples: 32.0b2, 38.0.5b2, 32.0b10 or 32.0b10build1
+    release: /^(\d+\.)+\d$/,        // Examples: 31.0 or 32.0.1
```

Comment:
At some point `REGEXES.beta` and `REGEXES.release` won't have a reason to exist. Information will be inferred from the other regexes